src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -328,15 +336,15 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
 
               <TokenExtras>
                 {amountValue && <StyledText>{formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
-                {yieldPercent && (
-                  <StyledText>
-                    {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
-                  </StyledText>
-                )}
               </TokenExtras>
             </TokenData>
           )}
         </TokenInfo>
+        {listItems?.length > 1 && (
+          <ZapMessageContainer onClick={listItems?.length > 1 ? openSearchList : undefined}>
```

Comment:
We already checked for this condition in the previous line, so we can safely assume that at this point `listItems.length > 1`
```suggestion
          <ZapMessageContainer onClick={openSearchList}>
```